clang/include/clang/Lex/Token.h

```diff
@@ -13,6 +13,7 @@
 #ifndef LLVM_CLANG_LEX_TOKEN_H
 #define LLVM_CLANG_LEX_TOKEN_H
 
+#include "clang/Basic/LangOptions.h"
```

Comment:
I think we can get by with a forward-declaration of `LangOptions` instead of the include here?